src/LucidServiceProvider.php

```diff
@@ -24,8 +24,19 @@ class LucidServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        if (!$this->app->routesAreCached()) {
-            require_once __DIR__.'/Http/routes.php';
+        $configPath = __DIR__ . '/../config/lucid.php';
+        $this->publishes([$configPath => $this->getConfigPath()], 'config');
+
+        $dashboard_enabled = $this->app['config']->get('lucid.dashboard');
+
+        if ($dashboard_enabled === null) {
+            $dashboard_enabled = $this->app['config']->get('app.debug');
+        }
+
+        if ($dashboard_enabled === true) {
```

Comment:
If you mean PSR-1 there is no recommendation about camel case or underscore names for variables but as whole package use camel case I'll change that. :-)